### PR TITLE
[BROWSEUI] Few corrections for de-DE translation

### PR DESCRIPTION
**Pull request by bodhi-baum**
Corrected some mistakes in dll/win32/browseui/lang/de-DE.rc

## Purpose

_Do a quick recap of your work here._

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

_Describe what you propose to change/add/fix with this pull request._

- 
- 

## TODO

_Use a TODO when your pull request is Work in Progress._

- [ ] 
- [ ]

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-german

**Comment by dreimer1986** [comment]
"Schaltfläche wechseln zu" vs "Schaltfläche Wechseln zu" is a good question... If this relates to a Button named "Wechseln zu" as it seems to be here, this would be wrong. Rest is fine.

File: dll/win32/browseui/lang/de-DE.rc [hunk 1]
```diff
@@ -31,7 +31,7 @@ BEGIN
         MENUITEM "S&ymbolleisten fixieren", IDM_TOOLBARS_LOCKTOOLBARS, MFT_STRING, MFS_ENABLED
         MENUITEM "&Anpassen...", IDM_TOOLBARS_CUSTOMIZE, MFT_STRING, MFS_ENABLED
         MENUITEM "Symboltitel", IDM_TOOLBARS_TEXTLABELS, MFT_STRING, MFS_ENABLED
-        MENUITEM "Schaltfläche Wechseln zu", IDM_TOOLBARS_GOBUTTON, MFT_STRING, MFS_ENABLED
+        MENUITEM "Schaltfläche wechseln zu", IDM_TOOLBARS_GOBUTTON, MFT_STRING, MFS_ENABLED
```
Comment (binarymaster) [inline dll/win32/browseui/lang/de-DE.rc:34]: Both translations are not clear, we discussed this in the chat just now, suggesting this translation:
```suggestion
        MENUITEM "Schaltfläche ""Wechseln zu""", IDM_TOOLBARS_GOBUTTON, MFT_STRING, MFS_ENABLED
```

**Comment by binarymaster** [comment]
Merged, thanks!

**Pull request merged**